OpenEdXMobile/src/main/java/org/edx/mobile/view/MyCoursesListFragment.kt

```diff
@@ -168,15 +169,10 @@ class MyCoursesListFragment : OfflineSupportBaseFragment(), RefreshListener {
                             }
                         }
                         HttpStatus.UPGRADE_REQUIRED -> {
-                            context?.let { context ->
-                                errorNotification.showError(
-                                    context,
-                                    it
-                                )
-                            }
+                            showError(EdxErrorState.State.UPDATE_APP)
```

Comment:
what will be the use case for other screens like, `My Programs`, and `Profile`.

Reply:
I checked on the current app and we are only showing this error state on MyCoursesScreen

Reply:
Please leave a message for Hilary, so if needed, we can tackle it in another ticket.